src/palimpzest/query/optimizer/cost_model.py

```diff
@@ -548,64 +468,36 @@ def _compute_operator_estimates(self) -> dict[str, Any] | None:
+                    cost_per_record = self._est_cost_per_record(op_df, model_name=model_name)
+                    input_tokens, output_tokens = self._est_tokens_per_record(op_df, model_name=model_name)
+                    selectivity = self._est_selectivity(self.sample_execution_data_df, op_df, model_name=model_name)
+                    quality = self._est_quality(op_df, model_name=model_name)
+
                     model_estimates = {
                         "time_per_record": time_per_record,
-                        "time_per_record_lower_bound": time_per_record_lb,
-                        "time_per_record_upper_bound": time_per_record_ub,
                         "cost_per_record": cost_per_record,
-                        "cost_per_record_lower_bound": cost_per_record_lb,
-                        "cost_per_record_upper_bound": cost_per_record_ub,
-                        "total_input_tokens": input_tokens_tup[0],
-                        "total_input_tokens_lower_bound": input_tokens_tup[1],
-                        "total_input_tokens_upper_bound": input_tokens_tup[2],
-                        "total_output_tokens": output_tokens_tup[0],
-                        "total_output_tokens_lower_bound": output_tokens_tup[1],
-                        "total_output_tokens_upper_bound": output_tokens_tup[2],
+                        "total_input_tokens": input_tokens,
+                        "total_output_tokens": output_tokens,
                         "selectivity": selectivity,
-                        "selectivity_lower_bound": selectivity_lb,
-                        "selectivity_upper_bound": selectivity_ub,
                         "quality": quality,
-                        "quality_lower_bound": quality_lb,
-                        "quality_upper_bound": quality_ub,
                     }
                     estimates[model_name] = model_estimates
 
             # TODO pre-compute lists of op_names in groups
             elif op_name in ["NonLLMFilter"]:
-                time_per_record, time_per_record_lb, time_per_record_ub = self._est_time_per_record(op_df)
-                selectivity, selectivity_lb, selectivity_ub = self._est_selectivity(self.sample_execution_data_df, op_df)
-                estimates = {
-                    "time_per_record": time_per_record,
-                    "time_per_record_lower_bound": time_per_record_lb,
-                    "time_per_record_upper_bound": time_per_record_ub,
-                    "selectivity": selectivity,
-                    "selectivity_lower_bound": selectivity_lb,
-                    "selectivity_upper_bound": selectivity_ub,
-                }
+                time_per_record = self._est_time_per_record(op_df)
+                selectivity = self._est_selectivity(self.sample_execution_data_df, op_df)
+                estimates = {"time_per_record": time_per_record, "selectivity": selectivity}
```

Comment:
I find selectivity confusing. Maybe, we should just call it fan_out?

Reply:
I agree, let's make this change universally in a separate PR once the other big-ticket items are merged.

Reply:
Created #151 for this